### PR TITLE
Summary table 

### PR DESCRIPTION
**Pull request by yisehak-awm**

### CODE REVIEW

File: src/assets/thirdPartyServices/mozi/gene_annotation_service/result/index.js [hunk 1]
```diff
@@ -137,6 +242,8 @@ const AnnotationResult = props => {
           handleClose={() => setTableShown(false)}
         />
       )}
+      {response && console.log("response.csv_files", response.csv_files)}
```
Comment (Vivek205) [inline src/assets/thirdPartyServices/mozi/gene_annotation_service/result/index.js]: remove this console.log

Comment (yisehak-awm) [reply src/assets/thirdPartyServices/mozi/gene_annotation_service/result/index.js]: Thank you, i have removed all console logs

**Comment by Vivek205** [comment]
@yisehak-awm , Is this change already merged to `snet/development`? If so, please provide the respective PRs link.

**Comment by yisehak-awm** [comment]
No it is not merged to snet/development

**Comment by Vivek205** [comment]
> No it is not merged to snet/development

Please integrate the changes to snet/development.

**Pull request closed without merging**